src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -224,4 +230,13 @@ public void setCompatibilityMode(final CompatibilityMode compatibilityMode) {
     public void setValidationStrategies(final List<ValidationStrategyConfiguration> validationStrategies) {
         this.validationStrategies = validationStrategies;
     }
+
+    @Nullable
```

Comment:
do you think it is fine that it will  be different from the way how get event-types/xxx work? It returns null properties.